econml/tests/test_bootstrap.py

```diff
@@ -270,8 +270,8 @@ def test_internal_options(self):
 
     def test_stratify(self):
         """Test that we can properly stratify by treatment"""
-        T = [1, 0, 1, 2, 0, 2]
-        Y = [1, 2, 3, 4, 5, 6]
+        T = np.array([1, 0, 1, 2, 0, 2])
+        Y = np.array([1, 2, 3, 4, 5, 6])
```

Comment:
Is this change necessary or just for consistency with X?  If the latter that's fine, but if the former then that's not ideal since it's nice to allow lists instead of proper numpy vectors when possible for convenience.